### PR TITLE
docs: Updates query to query and wait in `client_query_w_positional_params.py`

### PR DESCRIPTION
**Pull request by DevStephanie**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/client_query_w_positional_params.py [hunk 1]
```diff
@@ -35,7 +35,9 @@ def client_query_w_positional_params() -> None:
             bigquery.ScalarQueryParameter(None, "INT64", 250),
         ]
     )
-    query_job = client.query(query, job_config=job_config)  # Make an API request.
+    query_job = client.query_and_wait(
```
Comment (tswast) [inline samples/client_query_w_positional_params.py:38]: Please rename this variable, as the return value is no longer a QueryJob object. Instead it is a RowIterator object. "rows" or "results" could be a good name.

**Review by tswast — CHANGES_REQUESTED** [review]

File: samples/client_query_w_positional_params.py (hunk 1 above)
Comment (DevStephanie) [reply samples/client_query_w_positional_params.py:38]: Yes, sounds good. Will correct that now.

File: samples/client_query_w_positional_params.py [hunk 2]
```diff
@@ -35,7 +35,9 @@ def client_query_w_positional_params() -> None:
             bigquery.ScalarQueryParameter(None, "INT64", 250),
         ]
     )
-    query_job = client.query(query, job_config=job_config)  # Make an API request.
+    results = client.query_and_wait(
+        query, job_config=job_config
+    )  # Make an API request.
 
     for row in query_job:
```
Comment (tswast) [inline samples/client_query_w_positional_params.py:42]: This line needs to be updated too. The `query_job` variable doesn't exist anymore.

Comment (DevStephanie) [reply samples/client_query_w_positional_params.py:42]: Yes, will correct now.

**Review by tswast — APPROVED** [review]
Thanks!

**Pull request merged**